Source/Scene/ModelOutlineLoader.js

```diff
@@ -301,6 +308,11 @@ function addOutline(
         triangleIndices[i + 2] = copy;
       }
 
+      triangleIndexAccessorGltf.max[0] = Math.max(
+        triangleIndexAccessorGltf.max[0],
```

Comment:
The gltf 2.0 spec currently doesn't require min/max on all accessors, only those associated with a `POSITION` semantic, so at the moment this seems to be causing a `can't read from undefined` type error with models that worked prior to this change.

See attached for a Sandcastle, test model, and quick Node script I'm using as a test model generator:

[SteppyCube.zip](https://github.com/CesiumGS/cesium/files/4546018/SteppyCube.zip)

Reply:
Doh! I noticed and fixed that over 10 hours ago, and the git push has been sitting there waiting for my SSH password ever since. 😢
Fixed now!